docs/source/adding-yara-rules.rst

```diff
@@ -53,12 +105,13 @@ You can use these variables in your own rules to match or exclude certain filepa
           $evil and filepath matches /\/home\/*/
   }
 
+.. warning:: YARA analysis of archives `does not yet support external variables <https://github.com/BayshoreNetworks/yextend/issues/17>`_.
 
 .. _supported_yara_modules:
 
 Supported Modules
 -----------------
-BinaryAlert supports all of the default `YARA modules <http://yara.readthedocs.io/en/latest/modules.html>`_, including ELF, Math, Hash, and PE.
+BinaryAlert supports all of the default `YARA modules <http://yara.readthedocs.io/en/latest/modules.html>`_, including ELF, Math, Hash, and PE. Support for other modules is not planned at this time, but feel free to `open an issue <https://github.com/airbnb/binaryalert/issues>`_ if there's something you'd like to see.
```

Comment:
I may be policing too much here, but I'd avoid contractions in docs/formal writing (ie: `you'd` --> `you would`)

Reply:
I cannot believe I did not notice that - and I would totally agree

Reply:
lol I can't tell if you're being sarcastic 🤣

Reply:
Great suggestion! Done